src/System.Diagnostics.Process/tests/ProcessCollectionTests.cs

```diff
@@ -59,6 +59,7 @@ public void TestModuleCollectionBehavior()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "https://github.com/dotnet/corefx/issues/22174")]
```

Comment:
@joperezr we didn't identify what will be added and what is not applicable for uap yet